### PR TITLE
Issue 1 code-refactor

### LINKED ISSUE #7: Code refactor
1.add index to components & pages folder
2. refactor functions to take only necessary product data 
3. remove code for user signup alerts
4. remove unnececessary util functions for cart checkout
5. redirect to login from cart/wishlist
6. remove anchor tag
7. remove console log
8. remove inline css

### PR DESCRIPTION
**Pull request by aishwarya-bm**
minor issue on logout fixed
create index pages for components and pages 
remove unnecessary code for alerts
removed inline styles
removed all console.logs
replace empty cart/wishlist with images

### CODE REVIEW

**Review by sadathshariff — APPROVED** [review]
Minor fixes Looks Good

File: src/components/navpills/Navpills.jsx [hunk 1]
```diff
@@ -36,7 +36,9 @@ export default function Navpills() {
               <Link to="/cart">
                 <i className="fa fa-shopping-cart nav-btn"></i>
               </Link>
-              {cartSize > 0 && <span className="badge">{cartSize}</span>}
+              {isLoggedIn && cartSize > 0 && (
```
Comment (srejitk) [inline src/components/navpills/Navpills.jsx:39]: Good work checking for both conditions here

File: src/hooks/useAxios.jsx [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import { useState } from "react";
 import axios from "axios";
 
+// NOTE: This is not being used anywhere
```
Comment (srejitk) [inline src/hooks/useAxios.jsx:4]: Is this comment still valid? If so, you can remove this page, else delete the comment

File: src/components/index.jsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+export { RatingFilter } from "./filters/FilterBy/RatingFilter";
+export { SortFilter } from "./filters/FilterBy/SortFilter";
+export { FilterProducts } from "./filters/FilterProducts";
+export { Header } from "./header/Header";
+export { Navbrand } from "./navbrand/Navbrand";
+export { Navpills } from "./navpills/Navpills";
+export { Search } from "./search/Search";
+export { Signin } from "./signup/Signin";
+export { Signup } from "./signup/Signup";
+export { Toast } from "./toast/Toast";
```
Comment (therajatg) [inline src/components/index.jsx:14]: like that you are exporting all components from a single index file.

**Review by therajatg — APPROVED** [review]
liked that u removed export default and just kept export at the end in every file for consistency.  Liked the fact that u left ur job.

**Review by srejitk — APPROVED** [review]
Good work. I see that you've added a Toast for all user inputs/errors. 
Good Class naming convention

File: src/hooks/useAxios.jsx (hunk 2 above)
Comment (aishwarya-bm) [reply src/hooks/useAxios.jsx:4]: I might try and use it later maybe, so kept it here for future reference

**Comment by aishwarya-bm** [comment]
> liked that u removed export default and just kept export at the end in every file for consistency. Liked the fact that u left ur job.

Haha thanks Rajat :)

**Review by sadathshariff — COMMENTED** [review]
Overall the code looks clean and great work

**Pull request merged**